### DISCUSSION TITLE
Scaling out manager/search nodes

### DISCUSSION THREAD

**Discussion by csanate** [Security-Onion-Solutions/securityonion#3075] [Unsupported Versions] [answered]
Hello, 

I currently have a Forward Node and a Manager Search Node that comprise my deployment. I've searched over the SO2 documentation, but can't really find anything about scaling the cluster after the fact. Apologies in advance if I missed it. 

I want to migrate over to the recommended architecture. The reference image from the docs is where I'm trying to take this: 

![image](https://user-images.githubusercontent.com/38570031/108765958-1d883f80-751a-11eb-9d75-0590e6df3d76.png)

I just don't know if I need to tear down the entire thing and rebuild, or if there is hopefully a better way. 

Thanks!

**Comment by weslambert** [comment, marked as answer]
You can add another search node if you would like, or two more search nodes, then disable the search functionality on the manager, ultimately.

**Reply by csanate** [reply]
Thanks Wes, I'll give that a shot!
